### PR TITLE
Update of the readme to reflect current status

### PR DESCRIPTION
**Pull request by anagoessens**

### CODE REVIEW

File: apps/easypid/README.md [hunk 1]
```diff
@@ -4,25 +4,78 @@
...
+- 🟢 Set up biometrics
+- 🟢 History
+- 🟠 About the app
+- 🟢 Authentication using biometrics or PIN
+
+**Credential Management**
+- 🟢 Credential detail
+- 🟢 Delete QEAA
+- 🟢 Transaction history
+- 🟢 SD-JWT VC Type Metadata
```
Comment (TimoGlastra) [inline apps/easypid/README.md:28]: Type metadata is not supported yet

File: apps/easypid/README.md [hunk 2]
```diff
@@ -4,25 +4,78 @@
...
+- 🟢 Cross-device QR flow
+- 🟢 Same-device flow
+
+**Present attestations in-person**
+- 🔴 Query language
+- Android
+  - 🟢 Android-Android over NFC for device engagement
+- iOS
+  - 🔴 SD-JWT VC using OpenID4VP over BLE
+  - 🔴 mDOC using OpenID4VP over OpenID4VP
```
Comment (TimoGlastra) [inline apps/easypid/README.md:63]:
```suggestion
  - 🔴 mDOC over BLE
```

Comment (TimoGlastra) [reply apps/easypid/README.md:63]: These should also be added to ios

File: apps/easypid/README.md [hunk 3]
```diff
@@ -136,13 +191,25 @@ The following standards and specifications were implemented.
 - 🟢[OpenID for Verifiable Presentations - Draft 20](https://openid.net/specs/openid-4-verifiable-presentations-1_0-20.html)
 - 🟢 [SD-JWT VC - Draft 3](https://www.ietf.org/archive/id/draft-ietf-oauth-sd-jwt-vc-03.html)
 - 🟢 [Self-Issued OpenID Provider V2 - Draft 13](https://openid.net/specs/openid-connect-self-issued-v2-1_0-13.html)
-- 🟡 [ISO 18013-5](https://www.iso.org/standard/69084.html)
-  - Missing deviceSigned signing and verification
+- 🟢 [ISO 18013-5](https://www.iso.org/standard/69084.html)
 - 🟡 [ISO/IEC TS 18013-7 DTS Ballot Text](https://www.iso.org/standard/82772.html)
   - Missing JWT Secured Authorization Response Mode
```
Comment (TimoGlastra) [inline apps/easypid/README.md:196]: Supported now

File: apps/easypid/README.md [hunk 4]
```diff
@@ -136,13 +191,25 @@ The following standards and specifications were implemented.
...
 - 🟡 [High Assurance Interop Profile - Draft 0](https://openid.net/specs/openid4vc-high-assurance-interoperability-profile-sd-jwt-vc-1_0-00.html)
   - Missing verifier_attestation, `haip://` scheme and wallet attestation
-- 🔴 [OpenID Federation - Draft 34](https://openid.net/specs/openid-federation-1_0-34.html)
+- 🟠 [OpenID Federation - Draft 34](https://openid.net/specs/openid-federation-1_0-34.html)
+
+## Interoperability 
+
+The status of interoperability with other SPRIN-D teams
```
Comment (TimoGlastra) [inline apps/easypid/README.md:203]: I'm nit sure if we should make this public (it's already public now). 

But they were shared for the other funke teams, not necessarily for the public

File: apps/easypid/README.md (hunk 1 above)
Comment (anagoessens) [reply apps/easypid/README.md]: Is it then not a replacement for branding? Becasue we do support branding

Comment (TimoGlastra) [reply apps/easypid/README.md]: No it's an extension

**Review by TimoGlastra — APPROVED** [review]

**Pull request merged**